.github/ci.sh

```diff
@@ -109,10 +109,12 @@ bundle_files() {
 }
 
 sign() {
+  set +x
```

Comment:
This was done to avoid GPG-related information from leaking out to the build logs, similarly to how it's done for Crux [here](https://github.com/GaloisInc/crucible/commit/2157b1f92a3ea09dfae8014c874e90da2c8572d0). (Somewhat confusingly, `set +x` disables logging and `set -x` enables it.)

Reply:
Maybe this could use a comment to capture that intent.

Reply:
Good idea. I've added a comment above the `set +x`.